src/dev-app/select/select-demo.html

```diff
@@ -12,8 +12,8 @@
         <mat-label>Drink</mat-label>
         <mat-select [(ngModel)]="currentDrink" [required]="drinksRequired"
           [disabled]="drinksDisabled" #drinkControl="ngModel">
-          <mat-option>None</mat-option>
-          <mat-option *ngFor="let drink of drinks" [value]="drink.value" [disabled]="drink.disabled">
+          <mat-option value="" [disabled]="drinksOptionsDisabled">None</mat-option>
```

Comment:
Can you revert this change to `value=""`. This option is here so that it's easier to test options without a `value`.